### PR TITLE
Add minimal support for a SQL interface on Lasp.

### PR DESCRIPTION
**Pull request by cmeiklejohn**

### CODE REVIEW

**Comment by junghunyoo** [comment]
This looks good. If we can parameterise the identifier of the final output, it would be useful to share it with other processes.

**Comment by cmeiklejohn** [comment]
@junghunyoo I'm confused -- the create call returns the identifier to the user?

**Comment by junghunyoo** [comment]
In the ad counter test, we use the `<<"ads_with_contracts">>` for the id of shared object. I think it would be great if we could do similar behaviour with this SQL interface.

File: src/lasp_sql_materialized_view.erl [hunk 1]
```diff
@@ -24,12 +24,36 @@
...
+         get_value/2,
+         create_table_with_values/2]).
 
 -record(state, {output_id, set_id, projection_output_id, predicate_output_id}).
 
 -define(DEFAULT, <<"hi">>).
 
+%% @doc Insert in a SQL table the given rows.
+create_table_with_values(TableName, Args) when is_atom(TableName) ->
+    Name = list_to_binary(atom_to_list(TableName)),
```
Comment (cmeiklejohn) [inline src/lasp_sql_materialized_view.erl]: Maybe use the `generate_identifier` function I defined in this file for this?

**Pull request merged**